app/editor/src/features/content/list-view/ContentListView.tsx

```diff
@@ -50,6 +52,9 @@ const ContentListView: React.FC = () => {
   const [contentType, setContentType] = React.useState(formType ?? ContentTypeName.AudioVideo);
   const [isLoading, setIsLoading] = React.useState(false);
 
+  const [results, setResults] = React.useState<IContentSearchResult[]>([]);
```

Comment:
Why are you placing the search results into another state variable?  They already exist in the `content` variable.

Reply:
hmm, for some reason the content variable won't work in this case, or I might miss something here?